src/NServiceBus.AcceptanceTests/Sagas/Issue_2080.cs

```diff
@@ -0,0 +1,221 @@
+﻿namespace NServiceBus.AcceptanceTests.Sagas
+{
+    using System;
+    using EndpointTemplates;
+    using AcceptanceTesting;
+    using NUnit.Framework;
+    using Saga;
+
+    public class Issue_2080 : NServiceBusAcceptanceTest
```

Comment:
I would prefer this actually stated what it was testing rather than an issue number